tests/data/clientmodule_test.py

```diff
@@ -1,5 +1,6 @@
 """ docstring for file clientmodule.py """
 from data.suppliermodule_test import Interface, DoNothing, DoNothing2
+import toto
```

Comment:
This currently breaks the tests (not for `pyreverse`, but others, as this test data is also used for other test cases). Is this import intended?